lighthouse-core/audits/byte-efficiency/uses-long-cache-ttl.js

```diff
@@ -37,9 +37,11 @@ class CacheHeaders extends Audit {
    */
   static get defaultOptions() {
     return {
-      // see https://www.desmos.com/calculator/zokzso8umm
+      // 50th and 75th percentiles HTTPArchive -> 50 and 75
+      // https://bigquery.cloud.google.com/table/httparchive:lighthouse.2018_04_01_mobile?pli=1
```

Comment:
are there links for any of these in https://httparchive.org/reports? Nicer experience than just a link to the bigquery table, but not sure if @rviscomi has one for each LH audit

Reply:
I can link to the state of javascript/byte weight when we do that, but yeah if there are premade ones that'd be even better :D

Reply:
I also thought I remembered httparchive having # dom elements but I couldn't find the report where I expected and ended up querying ours instead

Reply:
The legacy site tracks average DOM element count: https://legacy.httparchive.org/trends.php#bytesHtmlDoc&numDomElements.

If there are any stats you'd like to see added to the redesigned site, please [file an issue](https://github.com/HTTPArchive/httparchive.org/issues) and consider contributing, I'd be happy to help with the setup.